lldb/source/Plugins/Process/gdb-remote/ProcessGDBRemote.h

```diff
@@ -111,7 +111,7 @@ class ProcessGDBRemote : public Process,
   // Process Control
   Status WillResume() override;
 
-  Status DoResume() override;
+  Status DoResume(lldb::RunDirection direction) override;
```

Comment:
We should use `DoResumeReverse()` in Process.h and this change will add a new interface definition.

Reply:
The decision was to use a direction flag.